package.json

```diff
@@ -12,15 +12,37 @@
     "pack": "npm run build && electron-builder --dir",
     "dist": "npm run build && electron-builder"
   },
-  "author": "",
-  "license": "ISC",
+  "author": {
```

Comment:
@phongcao @conrad465 if you'd like to be in this least, please feel free to add a commit to this branch with your info! 🚀

Reply:
Added mine. @conrad465 Please add yourself to the list.